apps/coscrad-frontend/src/components/nav-menu/nav-menu.spec.tsx

```diff
@@ -9,11 +9,11 @@ import { NavBar } from './nav-bar';
  * We probably also want Cypress tests to ensure the navigation works
  *
  */
-describe('NavBar', () => {
+describe('Nav Menu', () => {
     it('should render successfully', () => {
         const { baseElement } = renderWithProviders(
             <MemoryRouter>
-                <NavBar />
+                <NavMenuContainer />
```

Comment:
This reminds me we should really have a cypress test for interacting with the nav menu. Can you add a story for this?

Reply:
Story added